spec/sunomono_test.rb

```diff
@@ -0,0 +1,431 @@
+            not_to include("#{@project_name}")
+      end
+    end
+
+    context 'Should return the latest version of sunonomo' do
+      it 'Returns the latest version' do
+        system "sunomono version", :out => File::NULL
+        version = "#{Sunomono::VERSION}"
+
+        expect(version).
+            to end_with('0.2.0.pre')
+      end
+
+
+      it 'Should return the latest version of sunonomo using alias command' do
+        system "suno version", :out => File::NULL
+        version = "#{Sunomono::VERSION}"
+
+        expect(version).
+            to end_with('0.2.0.pre')
+      end
+    end
+
+    context 'Using I18n to translate files' do
+      it 'Generate an OS independent feature using I18n to portuguese language' do
+        system "suno new '#{@project_name}'", :out => File::NULL
+
+        feature_name = 'sunomono'
+
+        Dir.chdir(@project_name)
+
+        system "suno generate feature '#{feature_name}' --lang=pt", :out => File::NULL
+
+        Dir.chdir('../')
+
+        expect(File.readlines("#{@project_name}/features/#{feature_name}.feature")).
+            to include("# language: pt\n", "Funcionalidade: #{feature_name.capitalize} \n", "\n", "  Contexto:\n", "    # Insira os passos\n", "    \n", "  Cenário: Primeiro Cenario\n", "    # Insira os passos\n")
+        expect(File.readlines("#{@project_name}/features/step_definitions/#{feature_name}_steps.rb")).
+            to include("######### DADO #########\n", "\n", "######### QUANDO #########\n", "\n", "######### ENTãO #########")
+      end
+
+      it 'Generate an OS independent feature using english language' do
+        system "suno new '#{@project_name}'", :out => File::NULL
+
+        feature_name = 'sunomono'
+
+        Dir.chdir(@project_name)
+
+        system "suno generate feature '#{feature_name}'", :out => File::NULL
+
+        Dir.chdir('../')
+
+        expect(File.readlines("#{@project_name}/features/#{feature_name}.feature")).
+            to include("# language: en\n", "Feature: #{feature_name.capitalize} \n", "\n", "  Background:\n", "    # Insert steps\n", "    \n", "  Scenario: First Scenario\n", "    # Insert steps\n")
+        expect(File.readlines("#{@project_name}/features/step_definitions/#{feature_name}_steps.rb")).
+            to include("######### GIVEN #########\n", "\n", "######### WHEN #########\n", "\n", "######### THEN #########")
+      end
+    end
+
+    context 'This command should generates an OS independent feature(all files)' do
+      it 'Generates an OS independent feature' do
+        system "sunomono new '#{@project_name}'", :out => File::NULL
+
+        Dir.chdir(@project_name)
+
+        feature_name = 'rspec_sunomono'
+
+        system "sunomono generate feature '#{feature_name}'", :out => File::NULL
+
+        Dir.chdir('../')
+
+        expect(Dir.entries("#{@project_name}/features")).
+            to include('.', '..', 'android', 'ios', 'step_definitions', 'support', "#{feature_name}.feature")
```

Comment:
All these folder are not created by the g feature command and you tested it's creation on the first it.
I don see why re-check it here